cmake/FindMpris.cmake

```diff
@@ -0,0 +1,40 @@
+# Try to find mpris
```

Comment:
This name is a bit misleading. Strictly speaking, MPRIS is a DBus API for controlling music players https://specifications.freedesktop.org/mpris-spec/latest/ but what you are searching for here is a Qt5 wrapper around that API called QtMpris.

QTMPRIS_FOUND would be a better name